Tools/TestWebKitAPI/Tests/ios/DragAndDropTestsIOS.mm

```diff
@@ -2242,6 +2237,57 @@ static BOOL isCompletelyWhite(UIImage *image)
     EXPECT_WK_STREQ("com.pixar.universal-scene-description-mobile", [registeredTypes firstObject]);
 }
 
+#if ENABLE(MODEL_PROCESS)
+
+TEST(DragAndDropTests, CheckModelDragPreview)
+{
+    auto configuration = adoptNS([[WKWebViewConfiguration alloc] init]);
+    for (_WKFeature *feature in [WKPreferences _features]) {
+        if ([feature.key isEqualToString:@"ModelElementEnabled"] || [feature.key isEqualToString:@"ModelProcessEnabled"])
+            [[configuration preferences] _setEnabled:YES forFeature:feature];
+    }
+
+    auto messageHandler = adoptNS([[ModelLoadingMessageHandler alloc] init]);
+    [[configuration userContentController] addScriptMessageHandler:messageHandler.get() name:@"modelLoading"];
+
+    auto webView = adoptNS([[TestWKWebView alloc] initWithFrame:CGRectMake(0, 0, 320, 500) configuration:configuration.get()]);
```

Comment:
Nit - nowadays, we prefer using `RetainPtr` instead of `auto`, to make it clear where smart pointers have been deployed.